packages/harpguru-core/src/components/menu-face/menu-face.tsx

```diff
@@ -0,0 +1,30 @@
+import { View, StyleSheet } from 'react-native'
+import React from 'react'
+
+import { MenuCloseButton } from '../menu-close-button'
+import { getStaticMenuValues } from '../../utils'
+
+type MenuFaceProps = {
+  readonly children: React.ReactNode
+  readonly openCloseTapHandler: () => void
+}
+
+export const MenuFace = ({
+  children,
+  openCloseTapHandler,
+}: MenuFaceProps): React.ReactElement => {
+  const { labelProtrusion } = getStaticMenuValues()
+  const styles = StyleSheet.create({
```

Comment:
This could be stored on a `style` object by assigning with a dot property selector after the creation.

Reply:
Done